packages/storybook/.storybook/preview.tsx

```diff
@@ -90,7 +89,14 @@ export const parameters = {
         ],
       ]
     }
-  }
+  },
+  viewMode: 'docs',
+  previewTabs: {
+    'storybook/docs/panel': { index: -1 },
+    'canvas' : {
+      title: 'Try it out'
```

Comment:
(completely ignorable) suggestion: `Live Example` over Try it out?

Reply:
Sounds like a good discussion topic

Reply:
changing to "Preview"